Makefile

```diff
@@ -22,10 +22,27 @@ build.howitzer:
 
 # Deploy the resources used with the performance tests
 deploy.predictor:
-	kubectl apply -f ./perf_test/k8s/example-mnist-predictor.yaml
+#	kubectl apply -f ./perf_test/k8s/example-mnist-predictor.yaml
+#	oc apply -f ./perf_test/k8s/wml-serving-example-mnist-predictor.yaml
+
+	oc apply -f ./perf_test/k8s/keras-predictor.yaml
+	oc apply -f ./perf_test/k8s/pytorch-predictor.yaml
+	oc apply -f ./perf_test/k8s/lightgbm-predictor.yaml
+	oc apply -f ./perf_test/k8s/onnx-predictor.yaml
+	oc apply -f ./perf_test/k8s/tensorflow-predictor.yaml
+	oc apply -f ./perf_test/k8s/xgboost-predictor.yaml
+	oc apply -f ./perf_test/k8s/mlserver-sklearn-predictor.yaml
```

Comment:
@batuhandoverIBM I think we should use with `kubectl apply -f` here instead of `oc apply -f`.